Updated radio group topic (#409)
IgniteUI/indigo-design-docfx: en/components/radio-group.md

<img class="responsive-img" src="../images/radiogroup_demo.png" srcset="../images/radiogroup_demo@2x.png 2x" />

## Interaction State

> The Radio Group can be inserted in an enabled or disabled state, as well as you can set the state of each radio button in the group individually.
> 
> <img class="responsive-img" src="../images/radiogroup_enabled-state.png" srcset="../images/radiogroup_enabled-state@2x.png 2x" />
> <img class="responsive-img" src="../images/radiogroup_disabled-state.png" srcset="../images/radiogroup_disabled-state@2x.png 2x" />
> <img class="responsive-img" src="../images/radiogroup_enabled&disabled-state.png" srcset="../images/radiogroup_enabled&disabled-state@2x.png 2x" />
> 

Reviewer: ```suggestion
The Radio Group can be inserted in an enabled or disabled state.

<img class="responsive-img" src="../images/radiogroup_enabled-state.png" srcset="../images/radiogroup_enabled-state@2x.png 2x" />
<img class="responsive-img" src="../images/radiogroup_disabled-state.png" srcset="../images/radiogroup_disabled-state@2x.png 2x" />

You may also set the state of a radio button in the group to disabled to disallow user interaction with it.

<img class="responsive-img" src="../images/radiogroup_enabled&disabled-state.png" srcset="../images/radiogroup_enabled&disabled-state@2x.png 2x" />

```

## Usage

> When extending a Radio Group with additional items, make sure that they are all vertically left-aligned in a single column. Avoid layout in multiple columns, as well as situations with more than one Radio on at a time.

Reviewer: ```suggestion
When extending a Radio Group with additional items, make sure that they are aligned consistently and form a single column. Avoid layout in multiple columns, as well as situations with more than one Radio element with on state at a time.
```
